modules/end-user-guide/pages/selecting-an-in-browser-ide-for-one-new-workspace.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: PROCEDURE
+:description: Selecting an in-browser IDE for one new workspace
+:keywords: select-IDE
+:navtitle: Selecting an in-browser IDE for one new workspace
+:page-aliases: selecting-an-ide-by-using-a-url-parameter.adoc
+
+[id="selecting-an-in-browser-ide-for-one-new-workspace_{context}"]
+= Selecting an in-browser IDE for one new workspace
+
+You can select your preferred in-browser IDE when using a URL for starting a new workspace. This way, each developer using {prod-short} can start a workspace with a clone of the same project repository and yet with the personal choice of the in-browser IDE.
+
+.Procedure
+
+. Include the URL parameter for the workspace IDE in the URL for starting a new workspace. See xref:url-parameter-for-the-workspace-ide.adoc[].
```

Comment:
Is it literally a special start-a-new-workspace The wording is a bit confusing but I'm not sure how to make it better.

Reply:
The wording is very use-case specific:

_Working with Che in your browser involves multiple URLs:_
    _The URL of your organization’s Che instance, used as part of all the following URLs_
    _The URL of the Workspaces page of your Che dashboard with the workspace control panel_
    _**The URLs for starting a new workspace**_
    _The URLs of your workspaces in use_

Source: https://www.eclipse.org/che/docs/stable/end-user-guide/starting-a-new-workspace-with-a-clone-of-a-git-repository/